thirdeye/thirdeye-frontend/app/pods/rootcause/controller.js

```diff
@@ -0,0 +1,433 @@
+import Ember from 'ember';
+import { makeIterable, filterObject } from '../../helpers/utils';
+
+//
+// Config
+//
+
+const EVENT_TABLE_COLUMNS = [
+  {
+    template: 'custom/checkbox',
+    useFilter: false,
+    mayBeHidden: false,
+    className: 'events-table__column--checkbox'
+  },
+  {
+    propertyName: 'label',
+    title: 'Event Name',
+    className: 'events-table__column'
+  },
+  {
+    propertyName: 'eventType',
+    title: 'Type',
+    filterWithSelect: true,
+    sortFilterOptions: true,
+    className: 'events-table__column--compact'
+  },
+  {
+    propertyName: 'start',
+    title: 'Start',
+    className: 'events-table__column--compact',
+    disableFiltering: true
+  },
+  {
+    propertyName: 'end',
+    title: 'End',
+    className: 'events-table__column--compact',
+    disableFiltering: true
+  }
+];
```

Comment:
Don't forget to put this config in a separate file under /app/mocks and import it. This makes the file a lot less long and more readable.

Reply:
fixed